org.freedesktop.Sdk.Extension.rust-stable.appdata.xml

```diff
@@ -12,6 +12,7 @@
   <translation/>
   <update_contact>danigm@gnome.org</update_contact>
   <releases>
+    <release version="2021-01-11" date="2021-01-11"/>
```

Comment:
@bilelmoussaoui @Hofer-Julian Looks like the rust-analyzer version leaked to rust appdata.

Reply:
Ah, not great :/

Reply:
I guess that f-e-d-c assumes the last source to be the primary one and uses it's version. I'll look into improving this behavior.
In the meantime, probably merging rust-analyzer module with rust and making it first will help?